examples/todo_example/server/plugin.ts

```diff
@@ -0,0 +1,368 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type {
+  CoreSetup,
+  CoreStart,
+  Plugin,
+  PluginInitializerContext,
+  SavedObject,
+} from '@kbn/core/server';
+import type {
+  SavedObjectModelTransformationDoc,
+  SavedObjectsModelVersionMap,
+} from '@kbn/core-saved-objects-server';
+import { i18n } from '@kbn/i18n';
+import { schema } from '@kbn/config-schema';
+import type { Todo } from '../common';
+
+const savedObjectToTodo = (so: SavedObject<Omit<Todo, 'id'>>) => ({
+  id: so.id,
+  ...so.attributes,
+});
+
+const MODEL_VERSIONS: SavedObjectsModelVersionMap = {
```

Comment:
Just wanted to make sure it's clear why/when you would use these `backfill` transforms (since you wouldn't use them when first introducing a new saved object type). Was there a previous iteration of this PR that introduced the saved object without the `priority` field?